docs/06-concepts/11-authentication/04-providers/06-custom-providers.md

```diff
@@ -113,25 +113,53 @@ class MyAuthenticationEndpoint extends Endpoint {
 
 The example above shows how to create an `AuthenticationResponse` with the auth token and user information.
 
-### Remove auth token
+## Managing Authentication Keys and Signing Out Users
 
-Signing out a user on all devices is made simple with the `signOutUser` method in the `UserAuthentication` class. The method removes all auth tokens associated with the user.
+Serverpod provides built-in methods for managing user authentication across multiple devices. Below are the key functions for signing out users and revoking authentication keys. It is crucial to use these methods to ensure that all required steps, such as notifying clients and clearing session data, are properly handled.
+
+### Revoking Authentication Keys (Per Device)
```

Comment:
Suggestion: Remove (Per Device) since we don't know the implementation of how this is used. Multiple devices might share the same authentication key.